fe/fe-core/src/main/java/com/starrocks/connector/ConnectorMetadatRequestContext.java

```diff
@@ -25,5 +26,17 @@ public void setTableVersionRange(TableVersionRange value) {
     public TableVersionRange getTableVersionRange() {
         return tableVersionRange;
     }
+
+    public void setQueryMVRewrite(boolean v) {
```

Comment:
do not introduce mv concepts to the Connector， I think `forceUseCache` is better